lib/testing/src/lib/core/actions/identity/query.service.ts

```diff
@@ -24,6 +24,7 @@ export type TaskStatus = 'COMPLETED' | 'CREATED' | 'ASSIGNED' | 'SUSPENDED' | 'C
 export class QueryService {
 
     api: ApiService;
+    retryCount = 15;
```

Comment:
remove this, shared state that can affect parallel calls

Reply:
Sure